great_expectations/datasource/fluent/spark_file_path_datasource.py

```diff
@@ -20,12 +21,17 @@
 class CSVAsset(_FilePathDataAsset):
     # Overridden inherited instance fields
     type: Literal["csv"] = "csv"
+    header: bool = False
+    infer_schema: bool = False
+
+    class Config:
+        extra = pydantic.Extra.forbid
 
     def _get_reader_method(self) -> str:
         return self.type
 
     def _get_reader_options_include(self) -> set[str] | None:
-        return {"header", "inferSchema"}
+        return {"header", "infer_schema"}
```

Comment:
@Kilo59 This may be the issue -- `pyspark` uses `inferSchema` (this comes from its `Scala` internal implementation roots).  Thanks.